### PR TITLE
c101007058

### PR DESCRIPTION
**Pull request by LogicalNonsense2051**
Added "Trickstar Live Stage".

### CODE REVIEW

File: SAST-JP/c101007058.lua [hunk 1]
```diff
@@ -0,0 +1,79 @@
+--Trickstar Live Stage
+--Logical Nonsense
+
+--Substitute ID
+local s,id=GetID()
+
+function c2819435.initial_effect(c)
```
Comment (AlphaKretin) [inline SAST-JP/c101007058.lua]: Using an ID here instead of s

File: SAST-JP/c101007058.lua [hunk 2]
```diff
@@ -0,0 +1,79 @@
+--Trickstar Live Stage
+--Logical Nonsense
```
Comment (AlphaKretin) [inline SAST-JP/c101007058.lua]: If this is based on the anime version, make sure you include the original credit here too (if it wasn't or it didn't have one, this is fine)

Comment (LogicalNonsense2051) [reply SAST-JP/c101007058.lua]: Yeah, I forgot to factored in anime cards as a reference when I was scripting this card. So it was from scrap.

File: SAST-JP/c101007058.lua (hunk 1 above)
Comment (LogicalNonsense2051) [reply SAST-JP/c101007058.lua]: Uhhh...about that...I deleted my fork. Oops.

Comment (AlphaKretin) [reply SAST-JP/c101007058.lua]: It's fine. I'd merge and fix it myself, but somehow there's a conflict, so instead I'll fix it locally and upload that copy

**Comment by AlphaKretin** [comment]
Reuploading myself to resolve conflict

**Pull request closed without merging**